packages/p2p/src/components/orders/orders.scss

```diff
@@ -96,4 +96,18 @@
             fill: var(--text-general);
         }
     }
+    &__table {
+        &_row {
+            cursor: pointer;
+            padding-left: 1rem;
+
+            &:hover {
+                background: var(--general-hover);
+            }
+        }
+        &_link {
```

Comment:
Should the `_` be a `-` instead?

Reply:
@mustofa-binary can't we remove the _link now? since the whole row order was changed to be clickable and we're supposed to remove the link on order id?

Reply:
yes, i have removed the link, its just that i want to make it red and bold styles, ill change the naming  for it

Reply:
I've changed the link to a span and updated the classname.